ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__amd64-nightly.yaml

```diff
@@ -63,6 +63,12 @@ releases:
       product: ocp
       stream: nightly
       version: "4.13"
+  source:
```

Comment:
openshift-extended-upgrade-pre is not referenced in openshift-e2e-test-qe, so we do not need the setting here.

Reply:
it has openshift-upgrade-qe-test-heterogeneous and cucushift-sdn-migration-ovn which have pre step, so keep it.